### DISCUSSION TITLE
刚刚突然刷新了网页，插件加载不出来了，有人有头绪嘛

### DISCUSSION THREAD

**Discussion by Clov3rBug** [the1812/Bilibili-Evolved#4589] [提问 / Q&A] [answered]
重新安装插件和重启都试过了，没有用，是不是b站换了新版页面的原因啊

**Comment by the1812** [comment]
~~之前有人提过, 在 #4577 (但是我没什么头绪~~
没事, 我看岔了

**Reply by Clov3rBug** [reply]
不是这个，是整个插件都没了

**Reply by Clov3rBug** [reply]
<img width="416" alt="Snipaste_2024-01-01_20-14-36" src="https://github.com/the1812/Bilibili-Evolved/assets/75650477/32d338c4-3b9b-4892-bd89-b28814d7c14b">
<img width="416" alt="Snipaste_2024-01-01_20-14-28" src="https://github.com/the1812/Bilibili-Evolved/assets/75650477/b7fa4a28-cb69-4aca-9409-26888f6a6a7a">
我看不太来，不知道能看出什么嘛

**Reply by the1812** [reply]
![Snipaste_2024-01-01_20-21-30](https://github.com/the1812/Bilibili-Evolved/assets/26504152/e7032c2f-7f81-4c6a-bb9a-93c7bb19aaf3)
这个能点开看看吗

**Reply by Clov3rBug** [reply]
![image](https://github.com/the1812/Bilibili-Evolved/assets/75650477/d7d4e2d0-a3d2-4e47-a8d5-89267a2a9ffb)

**Reply by the1812** [reply]
读取设置失败, 你重装确定删干净了? Tampermonkey 有个脚本回收站不清掉是删不干净的

**Reply by Clov3rBug** [reply]
我是一次浏览器突然把cpu占满了，然后我直接重启浏览器，然后发现插件加载不出来了
重新安装是直接在bilibili-evolved的页面找到安装然后点击，进入油猴的界面然后点了重新安装，其他的啥都没动
那我现在是不是需要把油猴重新安装一下哇

**Reply by the1812** [reply, marked as answer]
油猴里删掉, 脚本回收站清空, 然后再安装

**Reply by Clov3rBug** [reply]
哦哦，谢谢耐心解答，我试一下

**Reply by Clov3rBug** [reply]
我把脚本删除以后重新安装就成功了，真的非常感谢
